spec/lib/stylesheet/compiler_spec.rb

```diff
@@ -74,20 +74,6 @@
     end
   end
 
-  it "supports asset-url" do
-    css, _map = Stylesheet::Compiler.compile(".body{background-image: asset-url('/images/favicons/github.png');}", "test.scss")
-
-    expect(css).to include("url('/images/favicons/github.png')")
-    expect(css).not_to include('asset-url')
-  end
-
-  it "supports image-url" do
-    css, _map = Stylesheet::Compiler.compile(".body{background-image: image-url('/favicons/github.png');}", "test.scss")
-
-    expect(css).to include("url('/favicons/github.png')")
-    expect(css).not_to include('image-url')
-  end
-
```

Comment:
What's the context for this removal?

Reply:
The functions have been deprecated and removed.